be/src/runtime/tablet_writer_mgr.cpp

```diff
@@ -24,232 +24,17 @@
 #include "common/object_pool.h"
 #include "exec/tablet_info.h"
 #include "runtime/descriptors.h"
+#include "runtime/exec_env.h"
 #include "runtime/mem_tracker.h"
 #include "runtime/row_batch.h"
 #include "runtime/tuple_row.h"
 #include "service/backend_options.h"
-#include "util/bitmap.h"
 #include "util/stopwatch.hpp"
 #include "olap/delta_writer.h"
 #include "olap/lru_cache.h"
 
 namespace doris {
 
-// channel that process all data for this load
-class TabletsChannel {
```

Comment:
If not necessary to modify, please don't change these files to avoid Introducing new questions